var/spack/repos/builtin/packages/hdf/package.py

```diff
@@ -66,4 +65,5 @@ def install(self, spec, prefix):
         configure(*config_args)
 
         make()
-        make("install")
+        make('check')
```

Comment:
Ironically enough, `make check` just failed for me on another computer, so I'm about to wrap this in `self.run_tests`. Sometimes `make check` is good at finding bugs that are fixable and sometimes there's nothing we can do about a broken test suite.